packages/block-editor/src/store/reducer.js

```diff
@@ -1315,6 +1315,12 @@ export function automaticChangeStatus( state, action ) {
 			if ( state !== 'final' ) {
 				return state;
 			}
+
+			return;
+		// Undoing an automatic change should still be possible after mouse
+		// move.
+		case 'STOP_TYPING':
+			return state;
 	}
 
 	// Reset the state by default (for any action not handled).
```

Comment:
I think the root problem here is that this is dangerous in reducers in general. I'm pretty sure this error can happen again if we introduce new random actions. That said,  now we have an e2e test to be aware of it at least.

Reply:
Right, a newly introduced action might need to be added here, but it's more likely that it shouldn't be added here. If we do the opposite, it might be that we do the wrong action on `Backspace`, which I think is worse. I think it's better to whitelist the actions that should be ignored (which there are way less of).